storage/rocksdb/CMakeLists.txt

```diff
@@ -61,6 +61,11 @@ IF(CMAKE_COMPILER_IS_GNUCXX)
     # TODO(yzha): Fix warnings in MyRocks
     ADD_COMPILE_FLAGS(rdb_datadic.cc COMPILE_FLAGS "-Wno-class-memaccess")
   ENDIF()
+
+  MY_CHECK_CXX_COMPILER_FLAG("-Wno-deprecated-copy" HAVE_NO_DEPRECATED_COPY)
+  IF(HAVE_NO_DEPRECATED_COPY)
+    ADD_COMPILE_FLAGS(../../rocksdb/db/db_impl/db_impl.cc COMPILE_FLAGS "-Wno-deprecated-copy")
```

Comment:
What were these fixes for?

Reply:
This PR is created on top of https://github.com/facebook/mysql-5.6/pull/1127
It fixes gcc-9.3 warning:
```
In file included from /data/mysql-server/fb-8.0.17/rocksdb/file/random_access_file_reader.h:16,
                 from /data/mysql-server/fb-8.0.17/rocksdb/file/file_prefetch_buffer.h:14,
                 from /data/mysql-server/fb-8.0.17/rocksdb/table/format.h:13,
                 from /data/mysql-server/fb-8.0.17/rocksdb/table/internal_iterator.h:14,
                 from /data/mysql-server/fb-8.0.17/rocksdb/db/pinned_iterators_manager.h:12,
                 from /data/mysql-server/fb-8.0.17/rocksdb/db/range_tombstone_fragmenter.h:15,
                 from /data/mysql-server/fb-8.0.17/rocksdb/db/memtable.h:19,
                 from /data/mysql-server/fb-8.0.17/rocksdb/db/memtable_list.h:17,
                 from /data/mysql-server/fb-8.0.17/rocksdb/db/column_family.h:17,
                 from /data/mysql-server/fb-8.0.17/rocksdb/db/db_impl/db_impl.h:22,
                 from /data/mysql-server/fb-8.0.17/rocksdb/db/db_impl/db_impl.cc:9:
/data/mysql-server/fb-8.0.17/rocksdb/include/rocksdb/file_system.h:106:3: note: because ‘rocksdb::FileOptions’ has user-provided ‘rocksdb::FileOptions::FileOptions(const rocksdb::FileOptions&)’
  106 |   FileOptions(const FileOptions& opts)
```